Bugzilla/User/Setting.pm

```diff
@@ -384,6 +394,13 @@ Params:      C<$name> - string - the name of the new setting
 
 Returns:     a pointer to a hash of settings
 
+=item C<remove_setting($name, \@options)>
+
+Description: Removes an existing setting and the options.
+
+Params:      C<$name> - string - the name of the new setting
```

Comment:
Params:      C<$name> - string - the name of the setting to be removed

Reply:
Fixed!